docs/makeup-alert-dialog/index.css

```diff
@@ -5,308 +5,307 @@
-    --color-blue-2: #93c9ff;
-    --color-blue-3: #659eff;
-    --color-blue-4: #3665f3;
-    --color-blue-5: #382aef;
-    --color-blue-6: #2b0eaf;
-    --color-blue-7: #121258;
-    --color-green-1: #ccfdce;
-    --color-green-2: #9ef4a6;
-    --color-green-3: #5ee471;
-    --color-green-4: #28a443;
-    --color-green-5: #1bab49;
-    --color-green-6: #05823f;
-    --color-green-7: #07522c;
-    --color-yellow-1: #fcf2bd;
-    --color-yellow-2: #f7e376;
-    --color-yellow-3: #fbcd25;
-    --color-yellow-4: #f7b100;
-    --color-yellow-5: #e58c02;
-    --color-yellow-6: #aa5404;
-    --color-yellow-7: #592e13;
-    --color-teal-1: #c2f2ef;
-    --color-teal-2: #71e3e2;
-    --color-teal-3: #1dcbca;
-    --color-teal-4: #02a2ac;
-    --color-teal-5: #01718f;
-    --color-teal-6: #0e4a6c;
-    --color-teal-7: #003147;
-    --color-lime-1: #f4fabe;
-    --color-lime-2: #e9f577;
-    --color-lime-3: #c9e43b;
-    --color-lime-4: #accf02;
-    --color-lime-5: #86b300;
-    --color-lime-6: #4b7d06;
-    --color-lime-7: #364f03;
-    --color-ai-solid-green-strong: #4ee04b;
-    --color-ai-solid-green-subtle: #f1fdf1;
-    --color-ai-solid-green-subtle-dark: #112611;
-    --color-ai-solid-blue-strong: #0968f6;
-    --color-ai-solid-blue-subtle: #f0f6fe;
-    --color-ai-solid-blue-subtle-dark: #112c31;
-    --color-ai-solid-purple-strong: #993ee0;
-    --color-ai-solid-purple-subtle: #f9f3fd;
-    --color-ai-solid-purple-subtle-dark: #20172f;
-    --color-ai-solid-red-strong: #ff4242;
-    --color-ai-solid-red-subtle: #fff4f4;
-    --color-ai-solid-red-subtle-dark: #321919;
-    --color-ai-solid-yellow-strong: #ffd80e;
-    --opacity-50: 0.04;
-    --opacity-100: 0.08;
-    --opacity-150: 0.12;
-    --opacity-200: 0.16;
-    --font-size-10: 0.625rem;
-    --font-size-12: 0.75rem;
-    --font-size-14: 0.875rem;
-    --font-size-16: 1rem;
-    --font-size-18: 1.125rem;
-    --font-size-20: 1.25rem;
-    --font-size-24: 1.5rem;
-    --font-size-30: 1.875rem;
-    --font-size-36: 2.25rem;
-    --font-size-46: 2.875rem;
-    --font-size-64: 4rem;
-    --font-size-smallest: var(--font-size-10);
-    --font-size-small: var(--font-size-12);
-    --font-size-default: var(--font-size-14);
-    --font-size-medium: var(--font-size-16);
-    --font-size-large-1: var(--font-size-20);
-    --font-size-large-2: var(--font-size-24);
-    --font-size-giant-1: var(--font-size-30);
-    --font-size-giant-2: var(--font-size-36);
-    --font-size-giant-3: var(--font-size-46);
-    --font-size-giant-4: var(--font-size-64);
-    --font-weight-regular: 500;
-    --font-weight-bold: 700;
-    --spacing-25: 2px;
-    --spacing-50: 4px;
-    --spacing-75: 6px;
-    --spacing-100: 8px;
-    --spacing-125: 10px;
-    --spacing-150: 12px;
-    --spacing-200: 16px;
-    --spacing-250: 20px;
-    --spacing-300: 24px;
-    --spacing-400: 32px;
-    --spacing-450: 36px;
-    --spacing-600: 48px;
-    --spacing-700: 56px;
-    --spacing-800: 64px;
+  --border-radius-50: 8px;
```

Comment:
This is odd. Skin css dist files are 2 space indentation (skin uses prettier defaults also), so I have to assume the 4 indentation is coming from MiniCSSExtractPlugin or CSS Loader. Will take a look to see if it is configurable.

Reply:
Oh, my bad. I see the skin tokens file is generated with 4 space indentation. 

I think we should ignore these generated CSS files and I will fix it on the skin side.

Reply:
Totally fine; gonna omit these as well for now. In fact, since these are a little obscure, ensured the ignore file points out that only the deeply nested `*.css` files are ignored since we do have some apparently human-written `*.css` files.

Reply:
See https://github.com/makeup/makeup-js/pull/156/files#diff-b640b344ee7f3f03d2a443795a5d0708ef50e2e6e34214109ab2aad13ad6ba98R10